src/selectable-group.js

```diff
@@ -65,16 +72,25 @@ class SelectableGroup extends React.Component {
 	 * Called while moving the mouse with the button down. Changes the boundaries
 	 * of the selection box
 	 */
-	_openSelector (e) {		
+	_openSelector (e) {	
+		if(this.state.mouseMoveStarted) return;
+		this.state.mouseMoveStarted = true;
+
+		e = this._desktopEventCoords(e);
+
 	    const w = Math.abs(this._mouseDownData.initialW - e.pageX);
 	    const h = Math.abs(this._mouseDownData.initialH - e.pageY);
 
+	    var component = this;
+
 	    this.setState({
 	    	isBoxSelecting: true,
 	    	boxWidth: w,
 	    	boxHeight: h,
 	    	boxLeft: Math.min(e.pageX, this._mouseDownData.initialW),
 	    	boxTop: Math.min(e.pageY, this._mouseDownData.initialH)
+	    }, function(){
+	    	component.state.mouseMoveStarted = false;
```

Comment:
Rather than using introducing a new assignment of `component = this`, consider using an ES6 arrow function. Also, we shouldn't be mutating the `state` object directly.